docs/cli/kops.md

```diff
@@ -0,0 +1,42 @@
+## kops
+
+kops is kubernetes ops
+
+### Synopsis
+
+
+kops is kubernetes ops.
```

Comment:
Looks like these are generated from the `Short` and `Long` `cobra.Command` fields.